### ISSUE TITLE
Properties are not Supported For Custom Actions

### ISSUE THREAD

**Issue by daryllabar** [SimplerSoftware/XrmTSToolkit#5]
I created a custom VerifyAddress Action with a property, but it's not working.  

``` js
export class VerifyAddressResponse extends XrmTSToolkit.Soap.ExecuteResponse {

    /**
     * Action Response Stringified JSON
     */
    Response: string;

    private _responseObj: VerifyAddressResponse;
    get responseObj(): VerifyAddressResponse {
        if (!this._responseObj) {
            this._responseObj = JSON.parse(this.Response);
        }
        return this._responseObj;
    }
}

```

The property is undefined.  Looks like the XrmTSToolkit just casts the Response to be the given type, but doesn't actually create one of that custom type.  What's the best way to get this to work?

**Comment by daryllabar** [comment]
I have resolved this by adding a CreateResponse method to the ExecuteRequest class, that then can be used to override the default instantiation.  I'm not sure what you're plan was for this respository.  Would you like to submit a pull request, or should I just keep my own fork of this?  I'd like to contribute to your base, but this is "yours" and if you want to keep it more basic, I'll keep working on my fork.

**Comment by StevenRasmussen** [comment]
Hi Daryl, thanks for your feedback so far! I definitely want to make XrmTSToolkit as flexible as possible and will certainly accept contributions where it makes sense.  Would you please submit a pull request for this so that I can take a look at what you've done here?  It sounds like this functionality should be included.  Thanks

**Comment by StevenRasmussen** [comment]
This was addressed in v0.8.5
